receiver/genericwebhookreceiver/Makefile

```diff
@@ -0,0 +1 @@
+include ../../Makefile.Common
```

Comment:
> I'm curious to see how the implementation of this receiver looks, but the name seems a bit unclear to me. Does the work "generic" really add anything here? Should it include the word "logs" or "logs", since we're modeling the data as logs? e.g. `webhooklogreceiver`

The use that I'm trying to convey is that the receiver is meant for capturing data from a wide range of push based data sources. The fact that the data ends up modeled as logs is more of a side affect than anything else, so I don't think that zeroing in on that clarifies the components use any further. 

I would be happy to drop the use of the word generic, it does seem a little clunky in there, but I don't think that including the word "log" clarifies the intended use.

Reply:
> The fact that the data ends up modeled as logs is more of a side affect than anything else 

This is an important point in my opinion.

We recently had a discussion about whether it is appropriate to use the logs data model to convey arbitrary data. The question was raised here: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/18526#issuecomment-1450459462

In that case, we were working with _alerts_. I argued that alerts are the same thing as events, which OTel official states share a data model with logs. Web hooks are essentially a protocol for communicating events, so it seems to me that this would properly be a logs (events) receiver. With that in mind, what if we call this `webhookevents`?

Reply:
> I argued that alerts are the same thing as events, which OTel official states share a data model with logs. Web hooks are essentially a protocol for communicating events, so it seems to me that this would properly be a logs (events) receiver. With that in mind, what if we call this `webhookevents`?

I appreciate your explaining the context/thought process here. I think I'm quite happy with the `webhookevents` name change